### DISCUSSION TITLE
请问大佬多音字有时候不准，怎么处理？比如：这是一场的的确确的冬雨

### DISCUSSION THREAD

**Discussion by stevin-dong** [fishaudio/Bert-VITS2#262] [General]
这是一场的的确确的冬雨
第一个“的”会发音为de，而不是di

**Comment by Stardust-minus** [comment]
> 这是一场的的确确的冬雨 第一个“的”会发音为de，而不是di

可以打印下g2p输出的拼音，八成是抽风了

**Reply by stevin-dong** [reply]
大佬，这种抽风的症状该给何种药？如何治疗呢？

**Reply by stevin-dong** [reply]
遇到不止一个多音字读错的情况了

**Reply by Stardust-minus** [reply]
<img width="585" alt="c8d589335e9cab7e742cc28646e64ab8" src="https://github.com/fishaudio/Bert-VITS2/assets/52093397/f1e93fa9-5a5a-402e-8a02-91eae54185e6">

**Reply by Stardust-minus** [reply]
已在ZH-Clap分支添加说明

**Reply by stevin-dong** [reply]
大佬，有没有从拼音(包含音调)生成汉字的包给推荐一个？因为要生成正确的谐音字需要这个

**Comment by jiangyuxiaoxiao** [comment]
考虑后续在`hiyoriUI`中添加手动标注
